lib/core/src/Cardano/Wallet/DB/Checkpoints/Model.hs

```diff
@@ -100,35 +120,74 @@ fromGenesis a = Checkpoints $ Map.singleton W.Origin a
 
 -- | Get the checkpoint with the largest 'SlotNo'.
 getLatest :: Checkpoints a -> (W.Slot, a)
-getLatest = from . Map.lookupMax . view #checkpoints 
+getLatest = fromMaybe err . Map.lookupMax . view #checkpoints
   where
-    from = fromMaybe (error "getLatest: there should always be at least a genesis checkpoint")
+    err = error
+        "getLatest: there should always be at least a genesis checkpoint"
 
 -- | Find the nearest 'Checkpoint' that is either at the given point or before.
 findNearestPoint :: Checkpoints a -> W.Slot -> Maybe W.Slot
 findNearestPoint m key = fst <$> Map.lookupLE key (view #checkpoints m)
 
 {-------------------------------------------------------------------------------
-    Delta type for Checkpoints
+        Delta type for Checkpoints
 -------------------------------------------------------------------------------}
 data DeltaCheckpoints a
     = PutCheckpoint W.Slot a
+    -- | Rolls back to the latest checkpoint at or before this slot.
     | RollbackTo W.Slot
-        -- Rolls back to the latest checkpoint at or before this slot.
-    | RestrictTo [W.Slot]
-        -- ^ Restrict to the intersection of this list with
-        -- the checkpoints that are already present.
-        -- The genesis checkpoint will always be present.
+    -- | Restrict to the intersection of this list with
+    -- the checkpoints that are already present.
+    -- The genesis checkpoint will always be present.
+    |    RestrictTo [W.Slot]
 
 instance Delta (DeltaCheckpoints a) where
     type Base (DeltaCheckpoints a) = Checkpoints a
     apply (PutCheckpoint pt a) = over #checkpoints $ Map.insert pt a
-    apply (RollbackTo pt) = over #checkpoints $
-        Map.filterWithKey (\k _ -> k <= pt)
+    apply (RollbackTo pt) =
+        over #checkpoints $
+            Map.filterWithKey (\k _ -> k <= pt)
     apply (RestrictTo pts) = over #checkpoints $ \m ->
-        Map.restrictKeys m $ Set.fromList (W.Origin:pts)
+        Map.restrictKeys m $ Set.fromList (W.Origin : pts)
```

Comment:
There seem to be several reformatting changes of code that doesn't obvious break the coding standards.

If it doesn't relate to the PR, would it be possible to remove changes like this from the PR?

Reply:
Done!